src/python/install.sh

```diff
@@ -26,7 +26,7 @@ CONFIGURE_JUPYTERLAB_ALLOW_ORIGIN="${CONFIGUREJUPYTERLABALLOWORIGIN:-""}"
 # alongside PYTHON_VERSION, but not set as default.
 ADDITIONAL_VERSIONS="${ADDITIONALVERSIONS:-""}"
 
-DEFAULT_UTILS=("pylint" "flake8" "autopep8" "black" "yapf" "mypy" "pydocstyle" "pycodestyle" "bandit" "pipenv" "virtualenv")
+DEFAULT_UTILS=("pylint" "flake8" "autopep8" "black" "mypy" "pipenv" "virtualenv")
```

Comment:
From https://github.com/devcontainers/images/pull/418#issuecomment-1426490478, I agree with @bamurtaugh thoughts of keeping these utils as is for now. Let us know what you think!